applications/giftless/values-roundtable-dev.yaml

```diff
@@ -0,0 +1,25 @@
+server:
+  debug: true
+ingress:
+  hostname:
+    readonly: "git-lfs-dev.lsst.cloud"
+    readwrite: "git-lfs-dev-rw.lsst.cloud"
+config:
+  readonly:
+    TRANSFER_ADAPTERS:
+      basic:
+        options:
+          storage_class: "giftless.storage.google_cloud:GoogleCloudStorage"
+          storage_options:
+            project_name: "plasma-geode-127520"
+            bucket_name: "rubin-gitlfs-experimental"
+            account_key_file: "/etc/secret/giftless-gcp-key.json"
```

Comment:
It's really annoying that this package is using the Google cloud library but doesn't support workload identity because upstream forces you to provide a key. If they would just let you not provide any credentials, workload identity would just work out of the box. It would probably be worth a PR to them at some point to fix that because we really do not want to ever use externalized credentials like this.

Reply:
When we go from roundtable-dev to a production environment, I want to have different read and write key files as well.  I agree that a PR to fix this to use workload identity would be a good idea.